besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -444,7 +444,8 @@ public Runner build() {
         RlpxConfiguration.create()
             .setBindHost(p2pListenInterface)
             .setBindPort(p2pListenPort)
-            .setMaxPeers(maxPeers)
+            .setPeerUpperBound(maxPeers)
+            .setPeerLowerBound(networkingConfiguration.getRlpx().getPeerLowerBound())
```

Comment:
Should we add a sanity check in BesuCommand.validateOptions() to ensure the lower bound <= upper bound

Reply:
done